tests/gcp/hooks/test_bigquery.py

```diff
@@ -732,6 +733,29 @@ def run_with_config(config):
         assert mocked_rwc.call_count == 1
 
 
+class TestLabelsInRunLoad(unittest.TestCase):
+    @mock.patch.object(hook.BigQueryBaseCursor, 'run_with_configuration')
+    def test_run_query_with_arg(self, mocked_rwc):
+        project_id = '12345'
+
+        def run_with_config(config):
+            self.assertEqual(
+                config['labels'], {'label1': 'test1', 'label2': 'test2',
+                                   'airflow-version': hook._AIRFLOW_VERSION}
+            )
+        mocked_rwc.side_effect = run_with_config
```

Comment:
Can you use assert_called_once_with instead creating special function?  This will make the code clearer.